components/sp-solutions/org.wso2.carbon.sp.solutions.apim.analytics/components/org.wso2.analytics.apim.rest.api.file/pom.xml

```diff
@@ -0,0 +1,118 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.wso2.analytics.solutions</groupId>
+        <artifactId>org.wso2.carbon.sp.solutions.apim.analytics</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>org.wso2.analytics.apim.rest.api.file</artifactId>
+    <packaging>bundle</packaging>
+    <modelVersion>4.0.0</modelVersion>
+    <name>Microgateway Analytics REST API</name>
+
+    <build>
+        <plugins>
+
+            <!--<plugin>-->
+                <!--<groupId>org.wso2.maven.plugins</groupId>-->
+                <!--<artifactId>swagger2msf4j-maven-plugin</artifactId>-->
+                <!--<version>1.0-SNAPSHOT</version>-->
+                <!--<configuration>-->
+                    <!--<inputSpec>${project.basedir}/src/main/resources/file-api.yaml</inputSpec>-->
+                <!--</configuration>-->
+            <!--</plugin>-->
```

Comment:
We can't uncomment this since we cannot do a release with a snapshot plugin version. We can't remove this since if the yaml is changed, we need this to auto-gen the code again.